CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+Hey, welcome to the party! 🎉
+
+Thank you so much to contribute to EasyMDE. 😘
+
+
+## Asking questions, suggesting wonderful ideas or reporting bugs
+
+You can [submit an issue️](https://github.com/Ionaru/easy-markdown-editor/issues/new) on this GitHub repository.
+
+
+## Coding
+
+### 📦 Prerequies
+
+You need node.js and npm.
+
+To install them on Debian-based systems:
+
+```bash
+curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+sudo apt-get install -y nodejs
+echo -e "nodejs version:\t$(nodejs -v) \nnpm version:\t$(npm -v)"
+# check that you have node.js and npm.
+```
+
+For other systems, please [read the official page](https://nodejs.org/en/download/).
+
+
+### 🏗️ Installation
+
+Here we go! 🤠 First, clone this repository:
+
+```bash
+git clone https://github.com/Ionaru/easy-markdown-editor.git
+```
+
+Then install the EasyMDE for development environment with npm:
+
+```bash
+npm install --save-dev
+```
+
+And check that everything is ok by running Gulp:
+
+```bash
+gulp
```

Comment:
There's no need to run `gulp` manually because `npm install` already runs Gulp after installing.